localization/pose_initializer/src/pose_initializer/pose_initializer_core.cpp

```diff
@@ -87,6 +91,9 @@ void PoseInitializer::on_initialize(
     if (ndt_) {
       pose = ndt_->align_pose(pose);
     }
+    if (yabloc_) {
```

Comment:
If both A and B are true, the estimation is performed on both A and B, but the result of A is discarded and the result of B is used.
 Is three any problem?

Reply:
Thanks for your reviewing!
I added `else if` to prioritize NDT if both initializer are activated in e71db0e .